feedback-module/src/components/Module.js

```diff
@@ -52,7 +56,7 @@ function Module({ pagetitle, endpoint, dir }) {
   const { t, i18n } = useTranslation();
   const en = i18n.getFixedT("en");
 
-  const { trackFutureResearch, pageTitleAsScreen, pageChange, moduleOnScreen } =
+  const { trackFormAction, pageTitleAsScreen, pageChange, moduleOnScreen } =
```

Comment:
Is there a reason you changed this event from only tracking whether the user signs up to be in the database to just tracking all calls in general?

Reply:
Yep, see this comment: https://github.com/nyc-cto/feedback-module/pull/59#discussion_r683788440